configs/xsolla.json

```
@@ -82,6 +82,7 @@
   "stop_urls": [
     "/search",
     "/changelog/tags",
+    "https://developers.xsolla.com(/[a-z][a-z])?/sdk/php/*",
```

```suggestion
    "/sdk/php/",
```

If the goal is to remove any pages under the `/sdk/php/`, this will do it!

---

We also have a locale identifier right after the domain like 
https://developers.xsolla.com/de/sdk/php/
https://developers.xsolla.com/ja/sdk/php/
https://developers.xsolla.com/ko/sdk/php/

Should it be `(/[a-z][a-z])?/sdk/php/` then?

---

`stop_urls` checks matching patterns, meaning as long as `/sdk/php/` is in the URL, it will be ignored

---

Got it, thanks!